conductr_cli/sandbox_run.py

```diff
@@ -27,23 +26,25 @@ def run(args):
     """`sandbox run` command"""
     is_conductr_v1 = major_version(args.image_version) == 1
     features = sandbox_features.collect_features(args.features, args.image_version)
+    sandbox = sandbox_run_docker if is_conductr_v1 else sandbox_run_jvm
 
-    if is_conductr_v1:
-        run_result = sandbox_run_docker.run(args, features)
-    else:
-        run_result = sandbox_run_jvm.run(args, features)
+    run_result = sandbox.run(args, features)
 
     is_started, wait_timeout = wait_for_start(args, run_result)
     if is_started:
         if not is_conductr_v1:
-            start_proxy(run_result.nr_of_proxy_instances)
+            feature_ports = []
```

Comment:
Not inside the `sandbox_run_jvm.run` method since the proxy can only be started after `wait_for_start` is completed and `is_started` is `True`.

Because the `features` also requires `wait_for_start` to be completed and `is_started` to be `True`, there's a need to `wait_for_start` for both `sandbox_run_docker` and `sandbox_run_jvm`.

The only way to fold this into `sandbox_run_jvm.run` method is to have the `wait_for_start` call done inside both `sandbox_run_docker` and `sandbox_run_jvm`. If we were to do this, we'd lose the DRY-ness of the `wait_for_start` and `features` start behaviour.

I'd rather not to do above since repeating `wait_for_start` and `features` start behaviour is less appealing than having the proxy steps inside `sandbox_run`.